config/consumers.py

```diff
@@ -1,25 +1,38 @@
+import logging
+
 from asgiref.sync import sync_to_async
 from channels.generic.websocket import AsyncJsonWebsocketConsumer
 
-from scram.route_manager.models import Entry
+from scram.route_manager.models import Entry, WebSocketSequenceElement
 
 
 class TranslatorConsumer(AsyncJsonWebsocketConsumer):
     async def connect(self):
+        logging.info("Translator connected")
         self.actiontype = self.scope["url_route"]["kwargs"]["actiontype"]
         self.translator_group = f"translator_{self.actiontype}"
 
         await self.channel_layer.group_add(self.translator_group, self.channel_name)
         await self.accept()
 
-        # TODO: Filter by our actiontype
+        # Filter WebSocketSequenceElements by actiontype
+        elements = await sync_to_async(list)(
+            WebSocketSequenceElement.objects.filter(action_type__name=self.actiontype).order_by("order_num")
+        )
+        if not elements:
+            logging.warning(f"No elements found for actiontype={self.actiontype}.")
```

Comment:
Might as well just return if no elements are found, rather than querying the routes.

Reply:
Sorry, can we do both? I think there's still value in the warning.

Reply:
Understood, added that back in.